src/components/CreateItem.jsx

```diff
@@ -1,18 +1,90 @@
-import React from 'react';
-import firebase from 'firebase/app';
+import React, { useState } from 'react';
+import db from '../lib/firebase';
+import './createItem.css';
 
 function CreateItem() {
-  const handleClick = async () => {
-    await firebase.firestore().collection('items').add({
-      name: 'A test product',
+  const itemObj = {
+    itemName: '',
+    frequency: 7,
+  };
+  const [item, setItem] = useState(itemObj);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const token = localStorage.getItem('token');
+
+    await db.collection('items').add({
```

Comment:
Curious why we're going to await this function, it doesn't look like anything relies on it completing.

Reply:
Yeah this is fair, I don't think i thought this through properly. I'll take another look.